src/mrpro/operators/_FourierOp.py

```diff
@@ -97,11 +101,14 @@ def get_traj(traj, dims):
 
             self._nufft_im_size = get_spatial_dims(recon_shape, self._nufft_dims)
             grid_size = [
-                int(s * os) for s, os in zip(self._nufft_im_size, get_spatial_dims(oversampling, self._nufft_dims))
+                int(s * os)
+                for s, os in zip(self._nufft_im_size, get_spatial_dims(oversampling, self._nufft_dims), strict=False)
```

Comment:
this should be strict=True imho

Reply:
changed in [c0e2d68](https://github.com/PTB-MR/mrpro/pull/193/commits/c0e2d68acdd11289bc3819520efef2bef172fa2c)